src/components/text.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactElement } from "react"
+
+interface TextProps {
+  as?:
+    | "h1"
+    | "h2"
+    | "p"
+    | "h3"
+    | "h3Link"
+    | "h2about"
+    | "pAbout"
+    | "pAuthor"
+    | "h2episode"
+    | "pEpisode"
+
+  children: string | any
+}
+
+export default function Text({
```

Comment:
Spread Attributes for all components here https://davidl.fr/pattern

Reply:
i think i fix it